src/oas-parsing/schema/oas-schema.ts

```diff
@@ -65,6 +65,13 @@ class OASSchema {
       options = { server, ...options };
     }
 
+    if (
+      options.parameters?.Accept !== undefined &&
+      options.parameters?.Accept.length > 0
+    ) {
+      options.responseContentType = options.parameters?.Accept;
```

Comment:
is there a reason to not just use options.parameters.Accept here since it is wrapped in an "if" statement that checks for undefined and zero length?